cisco_aci/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "cisco_aci",
+  "short_description": "Track Cisco ACI performance and usage.",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "guid": "8a20f56b-2e25-4a0b-a252-f5187dddeeef",
+  "public_title": "Datadog-CiscoACI Integration",
+  "categories":["cisco", "networking"],
+  "type": "check",
+  "doc_link": "https://docs.datadoghq.com/integrations/redisdb/",
+  "aliases": ["/integrations/cisco_aci/"],
+  "is_public": true,
+  "has_logo": false
+}
```

Comment:
Great job! 🎉

Are you going to make it a declarative tile? If so, I think you need to add a few fields so it can register as an integration that emits events. https://github.com/DataDog/devops/wiki/Declarative-Integrations#manifestjson

Reply:
I am not

Reply:
It's gonna be an old style one

Reply:
Why?

Reply:
So I can keep the dashboard and the code in one place. New style ones do not currently support dashboards. While I could make a new one and put the dashboard in dogweb, I'd rather keep them together for now and transfer it when we fix the dashboarding stuff